### PR TITLE
Zivid vtk vizualizer

### PR DESCRIPTION
**Pull request by SatjaSivcev**
Sample code added for point cloud visualization.

### CODE REVIEW

File: read_zdf_vis_3d.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    # Displaying the Depth Image
+    Z = xyz[:, :, 2]
+    plt.figure()
+    plt.imshow(Z, vmin=np.nanmin(Z), vmax=np.nanmax(Z), cmap="jet")
+    plt.colorbar()
+    plt.title("Depth map")
+
+
+if __name__ == "__main__":
+    # Before running this script, run '%gui qt'
```
Comment (trym-b) [inline read_zdf_vis_3d.py]: Have you tried running this through subprocess.Popen() ?

File: read_zdf_without_zivid.py [hunk 2]
```diff
@@ -24,14 +24,16 @@ def _main():
...
     plt.imshow(rgb)
-    plt.show()
+    plt.title("RGB image")
 
     # Displaying the Depth Image
     Z = xyz[:, :, 2]
+    plt.figure()
     plt.imshow(Z, vmin=np.nanmin(Z), vmax=np.nanmax(Z), cmap="jet")
     plt.colorbar()
-    plt.show()
```
Comment (trym-b) [inline read_zdf_without_zivid.py]: why is show removed?

File: read_zdf_vis_3d.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
+"""
+Import ZDF point cloud and visualize it using vtk_visualizer (point cloud) and
+matplotlib (RGB image and Depth map).
```
Comment (trym-b) [inline read_zdf_vis_3d.py]: This docstring is confusing to read. Terminology is not used accurately

File: read_zdf_without_zivid.py (hunk 2 above)
Comment (SatjaSivcev) [reply read_zdf_without_zivid.py]: It is not required. I think plt.imshow() already does that.

File: read_zdf_vis_3d.py (hunk 3 above)
Comment (SatjaSivcev) [reply read_zdf_vis_3d.py]: Changed to 'Import ZDF point cloud and visualize it.'
The libs are probably not too relevant for the description.

File: read_zdf_without_zivid.py (hunk 2 above)
Comment (SatjaSivcev) [reply read_zdf_without_zivid.py]: Actually, it was. This is why it didn't show up from the cmd. In Spyder it shows without it.

Comment (trym-b) [reply read_zdf_without_zivid.py]: So is this going to be reverted or not?

Comment (SatjaSivcev) [reply read_zdf_without_zivid.py]: we need to have plt.show() - the current version shows images.

File: read_zdf_vis_3d.py (hunk 1 above)
Comment (SatjaSivcev) [reply read_zdf_vis_3d.py]: I have and it didn't work. It is not required when running via cmd prompt.

Comment (SatjaSivcev) [reply read_zdf_vis_3d.py]: Should we close this conversation or do you think we should try to make it work in Spyder without having to separately run that line?

File: read_zdf_vis_3d.py [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    plt.show()
+
+    # Displaying the point cloud
+    plotxyzrgb(flattened_point_cloud)
+
+
+if __name__ == "__main__":
+    # If running the script from Spyder IDE, first run '%gui qt'
+    _main()
+    input("Press Enter to close...")
```
Comment (trym-b) [inline read_zdf_vis_3d.py]: move this into the function

**Pull request merged**